blocks/market-insight/market-insight.css

```diff
@@ -0,0 +1,252 @@
+.block.market-insight > ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1.25rem;
+}
+
+.block.market-insight > ul > li {
+  flex: 0 0 100%;
+  max-width: 100%;
+  background: var(--background-color-light-grey);
+  padding: 1.25rem;
+  border: 0.0625rem solid var(--border-color-gray);
+  height: 100%;
+  position: relative;
+}
+
+.block.market-insight .cards-title h3 {
+  font-size: var(--heading-font-size-s);
+  font-weight: var(--font-weight-semibold);
+  color: var(--color-gray-dark);
+  margin-bottom: 0.9375rem;
+  height: 2.875rem;
+  overflow: hidden;
+}
+
+.block.market-insight .cards-title h3 a {
+  font-size: var(--heading-font-size-s);
+  font-weight: var(--font-weight-semibold);
+  color: var(--color-gray-dark);
+  word-break: break-word;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  text-decoration: none;
+}
+
+.block.market-insight .cards-description {
+  line-height: 1.25rem;
+  color: var(--color-gray-dark);
+  word-break: break-word;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  display: -webkit-box;
+  height: 3.75rem;
+  -webkit-line-clamp: 3;
+  -webkit-box-orient: vertical;
+  margin-bottom: 3.75rem;
+}
+
+.block.market-insight .cards-description ul {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    vertical-align: baseline;
+}
+
+.block.market-insight .cards-description ul li {
+  text-align: justify;
+  list-style-type: disc;
+  list-style-position: inside;
+  color: var(--text-color-light);
+  margin-bottom: 1.25rem;
+  font: var(--font-weight-regular) var(--heading-font-size-ms) / var(--heading-font-size-m) Poppins, "Poppins Fallback", sans-serif;
+}
+
+.block.market-insight .cards-description li::marker {
+  color: var(--color-gray-dark);
+}
+
+.block.market-insight .cards-description p {
+  font: var(--font-weight-regular) var(--heading-font-size-ms) / var(--heading-font-size-m) Poppins, "Poppins Fallback", sans-serif;
+}
+
+.block.market-insight .cards-powerby {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  position: absolute;
+  width: 100%;
+  left: 0;
+  padding: 0.3125rem 1.25rem;
+  bottom: 0.625rem;
+  vertical-align: baseline;
+}
+
+.block.market-insight .cards-powerby p {
+  margin-bottom: 0;
+  font-size: var(--heading-font-size-xxs);
+  line-height: normal;
+  color: var(--text-color-gray);
+}
+
+.block.market-insight .cards-powerby .socialshare {
+  width: 4.625rem;
+}
+
+.block.market-insight .cards-powerby .socialshare button {
+  border: unset;
+  background-color: unset;
```

Comment:
`--background-color-unset` from global

Reply:
Okey, we have this kind of property. I'm a little bit confused that do/shall we have any other kinds of `unset` properties, like `--border-unset` `--margin-unset` , etc...